### DISCUSSION TITLE
Assistance with Setting Up FInOps Toolkit+Hub

### DISCUSSION THREAD

**Discussion by K7o7** [microsoft/finops-toolkit#1127] [Q&A] [answered]
Hi Guys, 
 
I have completed the deployment of finops hub. configured the scopes, setup the settings.json file and then moved into the ADF portion.

The current issues i am having are 
1. Setting up my first report in PowerBI. What is the best method here,? do I need to download the sample reports and then start connecting the data?. Is there a step by step guide /video on setting this up?
2. what is the best way to get this reports up, SAS token and then i can use this report to share with my wider teams.

thank you!

**Comment by soyroberto** [comment, marked as answer]
Have a look at this discussion https://github.com/microsoft/finops-toolkit/discussions/817#discussioncomment-11345210

**Reply by flanakin** [reply]
@K7o7 did this help? If not, let us know!

**Comment by K7o7** [comment]
Hi @flanakin  and @soyroberto,

Thanks for the info on the discussion,

Yes it did help! Manage to sort it out as well. Had some issues on the settings.json file as well.

Now i am configuring the reports on powerBI

Appreciate it!


Kesavan

Cloud Engineer | Cloud & Platform | JM IT KL

________________________________
